roles/kubernetes/preinstall/tasks/0081-ntp-configurations.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Ensure NTP package
+  package:
+    name:
+      - "{{ ntp_package }}"
+    state: present
+
+- name: Disable systemd-timesyncd
+  service:
+    name: systemd-timesyncd.service
+    enabled: false
+    state: stopped
+  failed_when: false
+
+- name: Set fact NTP settings
+  set_fact:
+    ntp_config_file: >-
+      {% if ntp_package == "ntp" -%}
+      /etc/ntp.conf
+      {%- elif ansible_os_family == 'RedHat' -%}
+      /etc/chrony.conf
+      {%- else -%}
+      /etc/chrony/chrony.conf
+      {%- endif -%}
+    ntp_service_name: >-
+      {% if ntp_package == "chrony" -%}
+      chronyd
+      {%- elif ansible_os_family == 'RedHat' -%}
```

Comment:
ditto

Reply:
Hi @oomichi , There are another special case.

The user specify the ntp_package=ntp in Centos7.

In Ubuntu/Debian/SUSE, use `systemctl start ntp` to start ntp service. Refer to
  -  https://askubuntu.com/questions/1009729/unable-to-start-ntpd-service
  - https://documentation.suse.com/sles/12-SP4/html/SLES-all/cha-netz-xntp.html
In Centos/RHEL, use  `systemctl start ntpd` to start ntp service. Refer to
  -  https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-configuring_ntp_using_ntpd